### PR TITLE
lightning: fix the behavior of "tidb.tls"

### PR DESCRIPTION
**Pull request by Frank945946**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Update tidb-lightning-configuration.md

Fix the behavior of the TiDB Lightning configuration item "tidb.tls"
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by lance6716 — COMMENTED** [review]
这个 PR 应当 pick 到除了 master 和 v8.1 的版本。master 和 v8.1 不要修改

File: tidb-lightning/tidb-lightning-configuration.md [hunk 1]
```diff
@@ -353,10 +353,10 @@ max-allowed-packet = 67_108_864
...
 # SQL 连接是否使用 TLS。可选值为：
 #  * ""            - 如果填充了 [tidb.security] 部分，则强制使用 TLS（与 "cluster" 情况相同），否则与 "false" 情况相同
-#  * "false"       - 禁用 TLS
-#  * "cluster"     - 强制使用 TLS 并使用 [tidb.security] 部分中指定的 CA 验证服务器的证书
-#  * "skip-verify" - 强制使用 TLS，但不验证服务器的证书（不安全！）
-#  * "preferred"   - 与 "skip-verify" 相同，但是如果服务器不支持 TLS，则会退回到未加密的连接
+#  * "false"       - 与取值为 "" 的行为相同。
+#  * "cluster"     - 强制使用 TLS 并使用 [tidb.security] 部分中指定的 CA 验证服务器的证书。
+#  * "skip-verify" - 强制使用 TLS，但不验证服务器的证书（不安全！）。如果服务器不支持 TLS，则会退回到未加密的连接。
+#  * "preferred"   - 与取值为 "skip-verify" 的行为相同。
```
Comment (Oreoxmt) [inline tidb-lightning/tidb-lightning-configuration.md:359]:
```suggestion
#  * "false"       - 与取值为 "" 的行为相同
#  * "cluster"     - 强制使用 TLS 并使用 [tidb.security] 部分中指定的 CA 验证服务器的证书
#  * "skip-verify" - 强制使用 TLS，但不验证服务器的证书（不安全！）。如果服务器不支持 TLS，则会退回到未加密的连接
#  * "preferred"   - 与取值为 "skip-verify" 的行为相同
```

**Review by Oreoxmt — APPROVED** [review]

File: tidb-lightning/tidb-lightning-configuration.md [hunk 2]
```diff
@@ -353,10 +353,10 @@ max-allowed-packet = 67_108_864
 
 # SQL 连接是否使用 TLS。可选值为：
 #  * ""            - 如果填充了 [tidb.security] 部分，则强制使用 TLS（与 "cluster" 情况相同），否则与 "false" 情况相同
```
Comment (Oreoxmt) [inline tidb-lightning/tidb-lightning-configuration.md:355]:
```suggestion
#  - ""：如果填充了 [tidb.security] 部分，则强制使用 TLS（与 "cluster" 情况相同）
```
"false" 与 "" 行为相同

File: tidb-lightning/tidb-lightning-configuration.md [hunk 3]
```diff
@@ -353,10 +353,10 @@ max-allowed-packet = 67_108_864
...
-#  * "false"       - 禁用 TLS
+#  * "false"       - 与取值为 "" 的行为相同
```
Comment (Oreoxmt) [inline tidb-lightning/tidb-lightning-configuration.md:356]:
```suggestion
#  - "false"：与取值为 "" 的行为相同
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 4]
```diff
@@ -353,10 +353,10 @@ max-allowed-packet = 67_108_864
...
 #  * "cluster"     - 强制使用 TLS 并使用 [tidb.security] 部分中指定的 CA 验证服务器的证书
```
Comment (Oreoxmt) [inline tidb-lightning/tidb-lightning-configuration.md:357]:
```suggestion
#  - "cluster"：强制使用 TLS 并使用 [tidb.security] 部分中指定的 CA 验证服务器的证书
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 5]
```diff
@@ -353,10 +353,10 @@ max-allowed-packet = 67_108_864
...
-#  * "skip-verify" - 强制使用 TLS，但不验证服务器的证书（不安全！）
-#  * "preferred"   - 与 "skip-verify" 相同，但是如果服务器不支持 TLS，则会退回到未加密的连接
+#  * "skip-verify" - 强制使用 TLS，但不验证服务器的证书（不安全！）。如果服务器不支持 TLS，则会退回到未加密的连接
+#  * "preferred"   - 与取值为 "skip-verify" 的行为相同
```
Comment (Oreoxmt) [inline tidb-lightning/tidb-lightning-configuration.md:359]:
```suggestion
#  - "skip-verify"：强制使用 TLS，但不验证服务器的证书（不安全！）。如果服务器不支持 TLS，则会退回到未加密的连接
#  - "preferred"：与取值为 "skip-verify" 的行为相同
```

File: tidb-lightning/tidb-lightning-configuration.md (hunk 2 above)
Comment (Frank945946) [reply tidb-lightning/tidb-lightning-configuration.md:355]: ，后面再加上 "否则会使用未加密的连接"

File: tidb-lightning/tidb-lightning-configuration.md (hunk 3 above)
Comment (Frank945946) [reply tidb-lightning/tidb-lightning-configuration.md:356]: LGTM

**Review by Oreoxmt — APPROVED** [review]

**Review by lance6716 — APPROVED** [review]

File: tidb-lightning/tidb-lightning-configuration.md [hunk 6]
```diff
@@ -352,11 +352,11 @@ sql-mode = "ONLY_FULL_GROUP_BY,NO_AUTO_CREATE_USER"
 max-allowed-packet = 67_108_864
 
 # SQL 连接是否使用 TLS。可选值为：
-#  * ""            - 如果填充了 [tidb.security] 部分，则强制使用 TLS（与 "cluster" 情况相同），否则与 "false" 情况相同
-#  * "false"       - 禁用 TLS
-#  * "cluster"     - 强制使用 TLS 并使用 [tidb.security] 部分中指定的 CA 验证服务器的证书
-#  * "skip-verify" - 强制使用 TLS，但不验证服务器的证书（不安全！）
-#  * "preferred"   - 与 "skip-verify" 相同，但是如果服务器不支持 TLS，则会退回到未加密的连接
+#  - ""：如果填充了 [tidb.security] 部分，则强制使用 TLS（与 "cluster" 情况相同），否则使用未加密的连接
```
Comment (qiancai) [inline tidb-lightning/tidb-lightning-configuration.md:355]:
```suggestion
#  - ""：如果配置了 [tidb.security] 部分的配置项，则强制使用 TLS（与取值为 "cluster" 的行为相同），否则使用未加密的连接
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: tidb-lightning/tidb-lightning-configuration.md [hunk 7]
```diff
@@ -352,11 +352,11 @@ sql-mode = "ONLY_FULL_GROUP_BY,NO_AUTO_CREATE_USER"
 max-allowed-packet = 67_108_864
 
 # SQL 连接是否使用 TLS。可选值为：
-#  * ""            - 如果填充了 [tidb.security] 部分，则强制使用 TLS（与 "cluster" 情况相同），否则与 "false" 情况相同
-#  * "false"       - 禁用 TLS
-#  * "cluster"     - 强制使用 TLS 并使用 [tidb.security] 部分中指定的 CA 验证服务器的证书
-#  * "skip-verify" - 强制使用 TLS，但不验证服务器的证书（不安全！）
-#  * "preferred"   - 与 "skip-verify" 相同，但是如果服务器不支持 TLS，则会退回到未加密的连接
+#  - ""：如果配置了 [tidb.security] 部分的配置项，则强制使用 TLS（与取值为 "cluster" 的行为相同），否则使用未加密的连接
```
Comment (qiancai) [inline tidb-lightning/tidb-lightning-configuration.md:355]:
```suggestion
#  - ""：如果配置了 [tidb.security] 部分的配置项，则 SQL 连接强制使用 TLS（与取值为 "cluster" 的行为相同），否则使用未加密的连接
```

**Pull request merged**